code/modules/surgery/organs/ass.dm

```diff
@@ -0,0 +1,101 @@
+/obj/item/organ/butt
+	name = "butt"
+	desc = "extremely treasured body part"
+	//alternate_worn_icon = 'icons/mob/head.dmi'
+	icon = 'icons/obj/butts.dmi'
+	icon_state = "butt"
+	inhand_icon_state = "butt"
+	zone = "groin"
+	slot = "butt"
+	throwforce = 5
+	throw_speed = 4
+	force = 5
+	embedding = list("embed_chance" = 5) // This is a joke
+	hitsound = 'sound/effects/fart.ogg'
+	body_parts_covered = HEAD
+	slot_flags = ITEM_SLOT_HEAD
+	var/loose = 0
+	var/pocket_storage_component_path = /datum/component/storage/concrete/pockets/butt
+
+/obj/item/organ/butt/Initialize()
+	. = ..()
+	if(ispath(pocket_storage_component_path))
+		LoadComponent(pocket_storage_component_path)
+
+/obj/item/organ/butt/xeno //XENOMORPH BUTTS ARE BEST BUTTS yes i agree
+	name = "alien butt"
+	desc = "best trophy ever"
+	icon_state = "xenobutt"
+	inhand_icon_state = "xenobutt"
+
+/obj/item/organ/butt/xeno/ComponentInitialize()
+	pocket_storage_component_path = /datum/component/storage/concrete/pockets/butt/xeno
+	. = ..()
+
+/obj/item/organ/butt/bluebutt // bluespace butts, science
+	name = "butt of holding"
+	desc = "This butt has bluespace properties, letting you store more items in it. Four tiny items, or two small ones, or one normal one can fit."
+	icon_state = "bluebutt"
+	inhand_icon_state = "bluebutt"
+	status = ORGAN_ROBOTIC
+
+/obj/item/organ/butt/bluebutt/ComponentInitialize()
+	pocket_storage_component_path = /datum/component/storage/concrete/pockets/butt/bluebutt
+	. = ..()
+
+/obj/item/organ/butt/Insert(mob/living/carbon/human/H, special = 0, drop_if_replaced = TRUE)
+	. = ..()
+	if(ispath(pocket_storage_component_path) && owner)
+		LoadComponent(pocket_storage_component_path)
+
+/obj/item/organ/butt/Remove(mob/living/carbon/M, special = 0)
+	var/turf/T = get_turf(M)
+	var/datum/component/storage/STR = GetComponent(/datum/component/storage)
+	if(STR)
+		var/list/STR_contents = STR.contents()
+		for(var/i in STR_contents)
+			var/obj/item/I = i
+			STR.remove_from_storage(I, T)
+
+	QDEL_NULL(STR)
+
+	. = ..()
+
+/obj/item/organ/butt/on_life()
+	var/datum/component/storage/STR = GetComponent(/datum/component/storage)
+	if(owner && STR)
+		var/list/STR_contents = STR.contents()
+		for(var/obj/item/I in STR_contents)
+			if(get_sharpness(I) == SHARP_EDGED || get_sharpness(I) == SHARP_POINTY)
+				owner.bleed(4)
+
+/obj/item/organ/butt/attackby(var/obj/item/W, mob/user as mob, params) // copypasting bot manufucturing process, im a lazy fuck
```

Comment:
```suggestion
/obj/item/organ/butt/attackby(obj/item/W, mob/living/user, params) // copypasting bot manufucturing process, im a lazy fuck
```
linter complaining about this, also I don't see a reason for the `mob/user as mob` so